File: C++ Code/Basics/Calculator/Simple Interest.cpp
#include<iostream>
using namespace std;
int main(){
    float p,i,y,SI;
    cout<<"enter the principle amount : ";
    cin>>p;
    cout<<"enter the interest rate : ";
    cin>>i;
    cout<<"enter the year : ";
    cin>>y;
    SI=p*i*y/100;
    cout<<endl;
    cout<<"Simple interest : "<<SI<<endl;
    cout<<"Total amount after "<<y<<" years = "<<p+SI;
    return 0;
}

File: Java Code/Basics/Calculator/SimpleInterest.java
import java.util.*;

public class SimpleInterest{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter the principal amount : ");
    float p=sc.nextFloat();
    System.out.print("Enter anual rate percent : ");
    float r=sc.nextFloat();
    System.out.print("Enter the time period : ");
    float t=sc.nextFloat();
    float SI=p*r*t/100;
    System.out.println("Simple Interest : "+SI);
    System.out.println("Total amount : "+(p+SI));
  }
}